src/sql/parts/jobManagement/views/stepsView.css

```diff
@@ -0,0 +1,52 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.steps-tree .list-row .status-icon {
+	height: 10px;
+	width: 10px;
+	display: inline-block;
+	margin-top: 4px;
+}
+
+.list-row .label {
+	padding-left: 10px;
+	display: flex;
+	text-align: center;
+}
+
+.passed {
```

Comment:
so we provide more specific class names and\or provide better scoping of the styles?  Some of these seem very likely to collide at some point.